### PR TITLE
Stickers: Hidden Library Panel

### LINKED ISSUE #6708: Hidden Sticker Panel for Template Creation
## Feature Description

After discussing MVP goals of stickers, it was decided we should add a hidden panel in the editor (behind a new feature flag) that when enabled shows available stickers that designers/RTcamp can use to avoid additional dev steps for preparing stickers for use. 

It is worth noting that stickers will _still_ need to be added to the codebase via PR. They will NOT be uploadable through the editor. For every template, there will need to be a dev task to add stickers for the designers/RTcamp to the codebase. It'll be a lot like the current process for adding assets to the code base. 

## Alternatives Considered

Without this, devs will have to manually add in stickers for designers/RTcamp through the editor within the template - combing through a lot of json to find the proper lines to replace.  The above drastically decreases go between and tedium. 

## Additional Context

This panel will **not** be external user facing.  

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Hidden panel that will be visible when feature flag is turned on that allows stickers to be dropped into template pages in the editor.

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Goal of this is to make an easy, non user facing, way to add stickers in the codebase to a story.

## Summary
Appends sticker buttons beneath the shapes panel behind a feature flag:

Screen Shot 2021-03-23 at 12 21 39 PM
Screen Shot 2021-03-23 at 12 21 15 PM

## Relevant Technical Choices
May not be pretty, but works. Not user facing so tried not to put too much effort into appearance and just make it work.

## To-do
NA

## User-facing changes
Nope

## Testing Instructions
go to experiments and enable hidden stickers. go to the editor in the shapes panel and see that you can add the sample sticker to the story.

### QA
NA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6708

### CODE REVIEW

File: includes/Experiments.php [hunk 1]
```diff
@@ -208,6 +208,17 @@ public function get_experiment_groups() {
...
 		return [
+			/**
+			 * Author: @littlemilkstudio
+			 * Issue: 6708
+			 * Creation date: 2021-03-23
+			 */
+			[
+				'name'        => 'enableHiddenStickers',
+				'label'       => __( 'Enable Hidden Stickers', 'web-stories' ),
+				'description' => __( 'Appends sticker buttons to the bottom off the shapes panel in library.', 'web-stories' ),
```
Comment (swissspidy) [inline includes/Experiments.php]: of\*

File: includes/Experiments.php [hunk 2]
```diff
@@ -208,6 +208,17 @@ public function get_experiment_groups() {
...
 	public function get_experiments() {
 		return [
+			/**
+			 * Author: @littlemilkstudio
+			 * Issue: 6708
+			 * Creation date: 2021-03-23
+			 */
+			[
+				'name'        => 'enableHiddenStickers',
+				'label'       => __( 'Enable Hidden Stickers', 'web-stories' ),
```
Comment (swissspidy) [inline includes/Experiments.php]: Perhaps strike the `hidden` from here so we can re-use that same flag later for the real thing. :-)

**Review by embarks — APPROVED** [review]
Thanks for the stickers Max :)

**Review by spacedmonkey — APPROVED** [review]
Looks good to me

**Pull request merged**